cypress/integration/commit/chart_toggle.ts

```diff
@@ -0,0 +1,45 @@
+const COMMITS_ROUTE = "/commits/evergreen";
```

Comment:
we should move this variable inside of a function. otherwise, i think this variable could be available to other cypress tests.